SamMou.Api.Graph/QueryObject.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using GraphQL;
+using GraphQL.Types;
+using SamMou.Api.Models;
+using SamMou.Api.Services.Interface;
+
+namespace SamMou.Api.GraphQL
+{
+    public class QueryObject : ObjectGraphType<object>
+    {
+        public QueryObject(IWeatherForecastService service)
+        {
+            try
+            {
+                Name = "Queries";
+                Description = "The base query for all the entities in our object graph.";
+
+                FieldAsync<WeatherForecastObject, WeatherForecast>(
+                    "singleweatherforecast",
+                    "Gets weatherforecast",
+                    new QueryArguments(
```

Comment:
can we specify the input? do you think that would be clearer?
```suggestion
                    arguments: new QueryArguments(
```

Reply:
ok sure